src/components/Header.jsx

```diff
@@ -36,6 +35,7 @@ const Header = (props) => {
         extra={(
           <Space size='large'>
             <Space>
+              {renderDomainSpecificContent('Courses')}
```

Comment:
maybe this should be written so that it is not so explicitly a location to put an external link to courses in case another party wants to use it for a different purpose. Something like `'OptionalFirstHeaderButton'` which happens to be for courses in the case of Biomage?

Reply:
right, `'Courses'` only makes sense in the context of the biomage deployment

Reply:
I renamed it to `'HeaderExtraButton'`